tsconfig-strict.json

```diff
@@ -98,8 +98,10 @@
     "extensions/interactions/AlgebraicExpressionInput/directives/algebraic-expression-input-rules.service.spec.ts",
     "extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.ts",
     "extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.spec.ts",
+    "extensions/interactions/LogicProof/directives/logic-proof-rules.service.ts",
+    "extensions/interactions/LogicProof/directives/logic-proof-rules.service.spec.ts",
     "core/templates/domain/classifier/classifier.model.ts",
-    "core/templates/domain/classifier/classifier.model.spec.ts"
+    "core/templates/domain/classifier/classifier.model.spec.ts",
```

Comment:
Please try to keep this list sorted, so move these two lines up before `core/templates/domain/creator_dashboard/creator-dashboard-stats.model.ts`

Reply:
Changes have been made, facing some failing frontend tests.

Reply:
Done